src/components/gauge/gauge.tsx

```diff
@@ -0,0 +1,373 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Listen,
+  Prop,
+  State,
+  h
+} from "@stencil/core";
+
+import { Component as GxComponent } from "../common/interfaces";
+
+@Component({
+  shadow: false,
+  styleUrl: "gauge.scss",
+  tag: "gx-gauge"
+})
+export class Gauge implements GxComponent {
+  @Element() element: HTMLGxGaugeElement;
+
+  /**
+   * The `gxGaugeDidLoad` event is triggered when component has been rendered completely.
+   */
+  @Event() gxGaugeDidLoad: EventEmitter;
+
+  /**
+   * Property of type Style.
```

Comment:
Not sure what this means. I'd remove it, the next line correctly expresses what the property is doing.

Reply:
The purpose of that line is to explain the type of impact of that property. That property makes visuals/styling changes.
BTW, that is from a previous version of the control. Removed in 89570cc